api/app/wildfire_one/schema_parsers.py

```diff
@@ -128,35 +130,26 @@ def generate_station_daily(raw_daily, station: WFWXWeatherStation, fuel_type: st
     """
     # pylint: disable=invalid-name
     # we use the fuel type lookup to get default values.
-    pc = FUEL_TYPE_LOOKUP[fuel_type]["PC"]
-    pdf = FUEL_TYPE_LOOKUP[fuel_type]["PDF"]
-    cbh = FUEL_TYPE_LOOKUP[fuel_type]["CBH"]
-    cfl = FUEL_TYPE_LOOKUP[fuel_type]["CFL"]
+    pc = FUEL_TYPE_DEFAULTS[fuel_type]["PC"]
+    pdf = FUEL_TYPE_DEFAULTS[fuel_type]["PDF"]
+    cbh = FUEL_TYPE_DEFAULTS[fuel_type]["CBH"]
+    cfl = FUEL_TYPE_DEFAULTS[fuel_type]["CFL"]
 
     isi = raw_daily.get('initialSpreadIndex', None)
     bui = raw_daily.get('buildUpIndex', None)
     ffmc = raw_daily.get('fineFuelMoistureCode', None)
-    fmc = cffdrs.foliar_moisture_content(station.lat, station.long, station.elevation, get_julian_date_now())
-
-    sfc = None
-    try:
-        sfc = cffdrs.surface_fuel_consumption(fuel_type, bui, ffmc, pc)
-    except cffdrs.CFFDRSException as exception:
-        logger.error(exception, exc_info=True)
+    fmc = cffdrs.foliar_moisture_content(
+        station.lat, station.long, station.elevation, get_julian_date_now())
+    sfc = cffdrs.surface_fuel_consumption(
```

Comment:
Removing the try block around `sfc` will break sometimes when WFWX doesn't have `bui`.

Idk if this is desired or not.

Reply:
In `cffdrs.surface_fuel_consumption()`, there's a check for `fuel_type`, `bui`, or `ffmc` being `None` and raises an exception if any of them are, since SFC can't be calculated if any of them are missing. So basically, Sy has just moved the error handling into the `cffdrs.py` file instead of in this file.

Reply:
Ah cool thanks, missed that.